import/popups/AuthPopupInterface.qml

```diff
@@ -13,6 +13,7 @@ import Sailfish.Silica 1.0
 
 UserPromptInterface {
     // inputs
+    property variant messageBundle
```

Comment:
Use var instead of variant as it's obsolete. Using variant may have serious performance implications.

Found this old commit:
https://github.com/sailfishos/sailfish-browser/commit/2eb40f46f92cfcd442556bac68e6b03561d05f18

[1]
https://doc.qt.io/qt-5/qml-variant.html

Reply:
Thanks for this. I've updated the variables to use `var` instead of `variant` (I wasn't aware of this, so useful for the future too).